src/main/java/com/laserfiche/api/client/apiserver/TokenClientImpl.java

```diff
@@ -11,9 +11,16 @@
 
 import java.util.Map;
 
+/**
+ * The Laserfiche Self-Hosted token route API client.
+ */
 public class TokenClientImpl extends OAuthClient implements TokenClient {
     private final String baseUrl;
 
+    /**
+     * Constructor for a TokenClient used to interact with the Laserfiche Self-Hosted token endpoint.
```

Comment:
I think it might be better to use a style similar to how JDK's classes are documented. 
I mean for constructors: https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#Random--

![image](https://user-images.githubusercontent.com/102711477/207429819-07c81986-228b-44aa-8545-207728f6c972.png)

Reply:
Updated to follow the JDK style for constructor javadoc